build/python_metadata.bzl

```diff
@@ -59,6 +59,9 @@ def make_bundle_version_info(versions):
         entry["feature_flags"] = [entry["flag"]]
         result[name] = entry
     dev = result["development"]
+
+    # Uncomment to test with development = 0.27.5
+    # dev["real_pyodide_version"] = "0.27.5"
```

Comment:
Remove?

Reply:
I want to keep this because it will make development faster for me.